domains/shivabajpai.json

```
@@ -0,0 +1,12 @@
+{
+  "description": "Portfolio website for Shiva Bajpai",
+  "repo": "https://github.com/Shiva-Bajpai/",
+  "owner": {
+    "username": "Shiva-Bajpai",
+    "email": "shivabajpai2006@gmail.com"
+  },
+  "record": {
+    "CNAME": "https://shivacodes.vercel.app"
```

```suggestion
    "CNAME": "shivacodes.vercel.app"
```
CNAME records cannot contain protocols (`https://`) or slashes (`/`).